### PR TITLE
image-layout: organize for anchors

### PR DESCRIPTION
**Pull request by vbatts**
Signed-off-by: Vincent Batts vbatts@hashbangbash.com

### CODE REVIEW

File: image-layout.md [hunk 1]
```diff
@@ -66,10 +63,26 @@ Those tags will often be represented in an image-layout repository with matching
 
 This illustrates the expected contents of a given ref, the manifest list it points to and the blobs the manifest references.
```
Comment (jonboulle) [inline image-layout.md]: could you move this below  Example Ref

Comment (vbatts) [inline image-layout.md]: done

File: image-layout.md [hunk 2]
```diff
@@ -64,12 +61,28 @@ For example, an image may have a tag for different versions or builds of the sof
...
 
-This illustrates the expected contents of a given ref, the manifest list it points to and the blobs the manifest references.
+### Example Ref
 
 ```
 $ cat ./refs/v1.0
 {"size": 4096, "digest": "sha256:e692418e4cbaf90ca69d05a66403747baa33ee08806650b51fab815ad7fc331f", "mediatype": "application/vnd.oci.image.manifest.list.v1+json"}
 ```
+
+This illustrates the expected contents of a given ref, the manifest list it points to and the blobs the manifest references.
```
Comment (wking) [inline image-layout.md]: Looking at this line more closely after it was [shifted][1], what is it actually saying?  “contents of a given ref” sounds like “a descriptor”, so ok.  “the manifest list it points to” might be the descriptor again.  But how is this illustrating “thte blobs the manifest references”?  Maybe we should drop this line entirely.

[1]: https://github.com/opencontainers/image-spec/pull/423#discussion_r85557101

Comment (vbatts) [inline image-layout.md]: Separate concern from this PR

Comment (wking) [inline image-layout.md]: Spun off into #427.

**Comment by jonboulle** [comment]
lgtm

**Comment by vbatts** [comment]
@opencontainers/image-spec-maintainers PTAL

**Comment by philips** [comment]
LGTM

**Comment by jonboulle** [comment]
sorry, rebase time

**Comment by vbatts** [comment]
updated. PTAL

**Comment by stevvooe** [comment]
LGTM

**Comment by jonboulle** [comment]
lgtm

**Comment by vbatts** [comment]
interesting. @stevvooe's LGTM didn't work? @caniszczyk ?

**Comment by stevvooe** [comment]
LGTM

**Comment by stevvooe** [comment]
@vbatts Rebase!

**Comment by vbatts** [comment]
rebased!

**Comment by jonboulle** [comment]
lgtm

**Comment by stevvooe** [comment]
LGTM

**Pull request merged**